tests/tests_suite_test.go

```diff
@@ -104,7 +107,10 @@ var _ = SynchronizedBeforeSuite(testsuite.SynchronizedBeforeTestSetup, testsuite
 var _ = SynchronizedAfterSuite(testsuite.AfterTestSuiteCleanup, testsuite.SynchronizedAfterTestSuiteCleanup)
 
 var _ = AfterEach(func() {
-	testCleanup()
+	skipGlobalCleanupLabelText := decorators.SkipGlobalCleanup[0]
+	if !slices.Contains(CurrentSpecReport().Labels(), skipGlobalCleanupLabelText) {
```

Comment:
check CurrentSpecReport().MatchesLabelFilter https://github.com/kubevirt/kubevirt/blob/65f436dc0a81e425358738cd644d87f293d310b2/vendor/github.com/onsi/ginkgo/v2/types/types.go#L315-L322

Reply:
pretty sure it's the labels on the spec

Reply:
Yeah bit painful but I don't see a way to get rid of it, youre right..